powerspikegg/computation_models/match/BUILD

```diff
@@ -41,6 +41,14 @@ py_library(
     srcs = ["train.py"],
     visibility = ["//visibility:public"],
     deps = [
-        "@pydep_tensorflow//:library",
+        ":tensorflow",
     ],
 )
+
+alias(
```

Comment:
I think you should move this into `//third_party/tensorflow/BUILD`. Also remove the old `//third_party/tensorflow` repository, so we do not have conflict with the new tensorflow dependencies.